### PR TITLE
chore: upgrade next 13 + react 18

### PR DESCRIPTION
**Pull request by alaister**

### CODE REVIEW

File: packages/config/code-hike.scss [hunk 1]
```diff
@@ -70,6 +70,9 @@
   //   font-family: Ubuntu, Droid Sans, -apple-system, BlinkMacSystemFont, Segoe WPC, Segoe UI,
   // sans-serif;
 }
+.ch-code-wrapper {
+  background-color: var(--ch-t-background) !important;
```
Comment (fsansalvadore) [inline packages/config/code-hike.scss:74]: @saltcod Not sure where `--ch-t-background` comes from.
I'm seeing the code snippets basically without a bg in the blog posts.

Screenshot 2023-10-26 at 16 06 17

Can I use the previous `--ch-16` here?

Comment (fsansalvadore) [reply packages/config/code-hike.scss:74]: With `--ch-16`:

Screenshot 2023-10-26 at 16 09 41

**Review by joshenlim — APPROVED** [review]

File: packages/ui/package.json [hunk 2]
```diff
@@ -18,59 +18,61 @@
...
+    "cmdk": "^0.2.0",
     "color": "^4.2.3",
     "colorjs.io": "^0.4.3",
     "common-tags": "^1.8.2",
     "date-fns": "^2.30.0",
     "formik": "^2.2.9",
     "lodash": "^4.17.21",
-    "lucide-react": "^0.124.0",
-    "openai": "^3.2.1",
+    "lottie-react": "^2.4.0",
```
Comment (MildTomato) [inline packages/ui/package.json:64]: random package added ?

Comment (alaister) [reply packages/ui/package.json:64]: some packages are used in the ui package and were missing

File: package.json [hunk 3]
```diff
@@ -34,26 +34,22 @@
...
-      "@code-hike/lighter": "^0.6.7"
-    }
-  },
   "devDependencies": {
-    "eslint": "^8.41.0",
+    "eslint": "^8.45.0",
     "eslint-config-custom": "*",
     "prettier": "^2.8.8",
     "prettier-plugin-sql-cst": "^0.5.0",
+    "sass": "^1.68.0",
```
Comment (MildTomato) [inline package.json:42]: random package added?

Comment (MildTomato) [reply package.json:42]: this is a weird one to add in the root

Comment (alaister) [reply package.json:42]: Next requires the sass package when using `.scss` files

File: packages/config/package.json [hunk 4]
```diff
@@ -11,11 +11,15 @@
...
     "deepmerge": "^4.2.2",
-    "eslint-config-next": "^13.3.0",
+    "eslint-config-next": "^13.5.3",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-react": "7.28.0",
     "tailwindcss-radix": "^2.0.0",
     "@tailwindcss/typography": "^0.5.9",
     "mini-svg-data-uri": "^1.4.3"
+  },
+  "devDependencies": {
```
Comment (MildTomato) [inline packages/config/package.json:21]: sure ?

File: packages/ui/package.json (hunk 2 above)
Comment (MildTomato) [reply packages/ui/package.json:64]: oh yea, i see it, Connecting component

**Review by MildTomato — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (MildTomato) [reply package.json:42]: but why in the root ?

Comment (MildTomato) [reply package.json:42]: nvm. just pointing it out

Comment (alaister) [reply package.json:42]: good point – I will move from root in a follow up PR

**Pull request merged**